XmlConfig/pom.xml

```
@@ -272,9 +272,9 @@
     <org.codehaus.mojo.xml-maven-plugin.version>1.0.2</org.codehaus.mojo.xml-maven-plugin.version>
     <com.mycila.license-maven-plugin.version>4.2.rc3</com.mycila.license-maven-plugin.version>
     <!-- == Dependency Versions == -->
-    <postgresql.version>42.5.1</postgresql.version>
+    <postgresql.version>42.7.1</postgresql.version>
     <ojdbc.version>21.8.0.0</ojdbc.version>
```

Correction omission?
```suggestion
    <ojdbc.version>23.3.0.23.09</ojdbc.version>
```

---

440ccfe06e4977a9eca7cdb3ec4f64df10f5d1c2
Upgraded ojdbc version.